Services/Dashboard/Achievements/classes/ilAchievements.php

```diff
@@ -95,10 +84,9 @@ public function isAnyActive(): bool
     }
 
     /**
-     * Get active services
      * @return int[]
      */
-    public function getActiveServices(): array
+    final public function getActiveServices(): array
     {
         return array_filter($this->services, function ($s) {
```

Comment:
As you changed something nearby, would you adjust this too?
```suggestion
        return array_filter($this->services, $this->isActive(...));
```
Please see: https://www.php.net/manual/en/functions.first_class_callable_syntax.php